README.md

```diff
@@ -1 +1,117 @@
 # Razeedash
+
+Razeedash is an app to manage deployments on Kubernetes
+
+## Requirements
+
+- Kubernetes CLI Tools
+- Kubernetes Cluster
+- [razeedash-api](https://github.com/razee-io/razeedash-api)
+- MongoDB
+- Register GitHub Application
+
+## Environment Varibles
+
+| Name | Value | Description |
+| ---- | ----- | ----------- |
+| OAUTH_SECRET_KEY | - | GitHub OAuth Secret Key |
+| GITHUB_URL | <https://github.com> | GitHub URL |
```

Comment:
Can you mark GITHUB_URL, GITHUB_API_URL, BUILD_ID and LAST_COMMIT_ID as optional?  Also, RAZEE_GHE_URL can be removed

Reply:
Updated